Revert "matches can be used with array as a parameter (#17010)" (#17143)
mdn/content: files/en-us/web/api/element/matches/index.md

---
{{APIRef("DOM")}}

The **`matches()`** method of the {{domxref("Element")}} interface tests whether the element would be selected by the specified [CSS selector](/en-US/docs/Learn/CSS/Building_blocks/Selectors).

## Syntax

```js
matches(selectorString)
```

### Parameters

- `selectors`

>   - : A string of valid [CSS selector](/en-US/docs/Learn/CSS/Building_blocks/Selectors) to test the {{domxref("Element")}} against.

Reviewer: ```suggestion
  - : A string containing valid [CSS selectors](/en-US/docs/Learn/CSS/Building_blocks/Selectors) to test the {{domxref("Element")}} against.
```

? "A string of selector" doesn't seem to be grammatical.

### Return value

`true` if the {{domxref("Element")}} matches the `selectors`. Otherwise, `false`.

### Exceptions

- `SyntaxError` {{domxref("DOMException")}}

>   - : Thrown if the `selectors` is not a valid CSS selector.

Reviewer: ```suggestion
  - : Thrown if `selectors` cannot be parsed as a CSS selector list.
```

?
